data/pathfinder/dreamscarred_press/path_of_war/pow_maneuvers.lst

```diff
@@ -1002,7 +1002,7 @@ Scarlet Riposte.MOD					DESC:Within the focus of the disciple's mind, pathways t
 Scything Strike.MOD					DESC:The principles of the Scarlet Throne teach its disciples that every swing must be made with precision and excellence; no movement is wasted and no attack should fruitless. The disciple may make a single attack against two adjacent enemies using the same attack roll and applying it to each target.|PRERULE:1,DisplayFullSpell
 Searing Break.MOD						DESC:By binding the light of the sun's rays into his weapons, the disciple of the Solar Wind may attack with the sun's burning heat. As a swift action, the initiator may add an additional 3d6 points of fire damage on all ranged attack for one round. This is a supernatural ability.|PRERULE:1,DisplayFullSpell
-Shards of Adamantine Strike.MOD			DESC:It is said that masters of the discipline are capable of using their hand in the manner of striking serpent and punch through solid stone as if it were straw, and disciples with this maneuver are those who can. Taking this training to the theater of war, a punch so potent that it can leave the foe sickened with pain is what this art form can deliver. The initiator makes an attack against the target creature and if successful, this attack inflicts an additional 12d6 points of damage that ignores the target's damage reduction or an object's hardness. If striking a living creature, the target must make a Fort save (DC 17 + primary initiator attribute modifier) or be nauseated with the pain of the strike for 1d4 rounds.|PRERULE:1,DisplayFullSpell
+Shards of Adamantine Strike.MOD			DESC:It is said that masters of the discipline are capable of using their hand in the manner of striking serpent and punch through solid stone as if it were straw, and disciples with this maneuver are those who can. Taking this training to theater of war, a punch so potent that it can leave the foe sickened with pain is what this art form can deliver. The initiator makes an attack against the target creature and if successful, this attack inflicts an additional 12d6 points of damage that ignores the target's damage reduction or an object's hardness. If striking a living creature, the target must make a Fort save (DC 17 + primary initiator attribute modifier) or be nauseated with the pain of the strike for 1d4 rounds.|PRERULE:1,DisplayFullSpell
```

Comment:
Ditto... looks like this file is removing "the" from in front of "theater". I would guess the search and replace method was looking for "the the" but wasn't using WHOLE word, so "the" in "theater" was caught.